### PR TITLE
added aid for transferring

### PR DESCRIPTION
**Pull request by abishakex**
added a line that could potentially help people from not erroring out while transferring files between devices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ The scratch storage is probably the more interesting of all the storage options.
 
 PACE recommends that you use Globus to transfer files to and from PACE. I have tried this before, but it is very buggy. I personally recommend using `scp` or `rsync` as ways to transfer files to and from PACE.
 
+Before you use the command to transfer files, make a new instance of the terminal while your main terminal is logged into pace, and then run the rysnc/scp commands on the new terminal.
```
Comment (guru-desh) [inline README.md]: Can you explain why this is needed? Why do you need another terminal on PACE open while running the rsync/scp commands?

Comment (guru-desh) [inline README.md]: Also -- you misspelled rsync as "rysnc"

Comment (guru-desh) [inline README.md]: Capitalize PACE

**Review by guru-desh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (abishakex) [reply README.md]: i have no idea ngl, i just did it because chad told me to and it worked 💀

Comment (guru-desh) [reply README.md]: You don't need to have another terminal. I've done rsync before without having a terminal connected to PACE.

**Comment by guru-desh** [comment]
Closed based on discussion with Chad

**Pull request closed without merging**